packages/terra-demographics-banner/src/DemographicsBanner.scss

```diff
@@ -0,0 +1,95 @@
+@import '~terra-mixins';
+@import './variables';
+@import './mixins';
+
+.terra-DemographicsBanner {
+  background: $demographics-background;
+  color: $demographics-foreground;
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 0.714em;
+  padding: 0.714em;
+
+  @media screen and (max-width: 767px) {
+    display: block;
+  }
+}
+
+.terra-DemographicsBanner--value {
+  display: inline-flex;
+
+  @if $terra-bidi {
+    [dir='ltr'] & {
+      margin-right: 0.357em;
```

Comment:
+1 for writing this without the mixin. The mixn has limitations with the sass `&` parent selector and advanced nested selectors.